sope-ldap/NGLdap/NGLdapConnection.m

```diff
@@ -194,10 +195,12 @@ - (NSException *)_exceptionForErrorCode:(int)_err
                            @"operation %@ failed: %@ (0x%X)",
                            _operation, ldapError, _err];
 
+  userInfo = [_ui mutableCopy];
```

Comment:
Hello @extrafu , I am new to sope code. 

I had the impression that the object was released with the dispatcher method pool at https://github.com/Zentyal/sope/blob/jag/exception-error-code/sope-appserver/NGObjWeb/SoObjects/SoObjectMethodDispatcher.m#L204
 . I am wrong?

Another related thing, in https://github.com/Zentyal/sope/blob/jag/exception-error-code/sope-ldap/NGLdap/NGLdapConnection.m#L303 the ui dictionary is released before raising the exception, in other similiar places it is not released. There is a reason for this or it is a error?.